.github/dependabot.yml

```diff
@@ -31,3 +31,13 @@ updates:
           - "golang.org/x/*"
     reviewers:
       - "github/codeql-go"
+
+  - package-ecosystem: "gomod"
+    directory: "go/ql/test"
+    schedule:
+      interval: "daily"
```

Comment:
Would it make sense to make this less often? [It seems](https://docs.github.com/en/code-security/dependabot/dependabot-version-updates/configuration-options-for-the-dependabot.yml-file#scheduleinterval) we could do monthly.

Reply:
That makes sense, although [according to the docs the `interval` setting doesn't affect security updates](https://docs.github.com/en/code-security/dependabot/dependabot-version-updates/configuration-options-for-the-dependabot.yml-file#configuration-options-for-the-dependabotyml-file). But I suppose we can still reduce it to `monthly` for the ordinary Dependabot updates since that should be a no-op when it happens anyway.

Reply:
```suggestion
      interval: "monthly"
```